HappyTrip.Reservation.System.Controller/Filters/DriversResultFilterAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿using AutoMapper;
+using HappyTrip.Reservation.System.Domain.Data.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HappyTrip.Reservation.System.Controller.Filters
+{
+    public class DriversResultFilterAttribute : ResultFilterAttribute
+    {
+        public override async Task OnResultExecutionAsync(ResultExecutingContext context, ResultExecutionDelegate next)
+        {
+            var resultFromAction = context.Result as ObjectResult;
+
+            if (resultFromAction?.Value == null || resultFromAction.StatusCode < 200 || resultFromAction.StatusCode >= 300)
```

Comment:
Does this code indicate the range of successful status codes from 200 to 300 `resultFromAction.StatusCode < 200 || resultFromAction.StatusCode >= 300`?

If so, should it be `resultFromAction.StatusCode >= 200 || resultFromAction.StatusCode <= 300` instead?